### DISCUSSION TITLE
Apache2

### DISCUSSION THREAD

**Discussion by StevenSoulet** [deployphp/deployer#3412] [Help needed]
Hello,
Is there a solution for APACHE regarding this link?
https://deployer.org/docs/7.x/avoid-php-fpm-reloading

When I do a deploy, I have to reload apache2 so that it takes into account the new folder for the current.

**Comment by antonmedv** [comment]
I guess something like this should be available. Maybe some configuration or module.

**Reply by StevenSoulet** [reply]
I looked on the internet and I can't find, so I ask here, if a solution had already been found.
